### DISCUSSION TITLE
My project does not deploy

### DISCUSSION THREAD

**Discussion by gromeronaranjo** [vercel/community#6074] [Help]
### Summary

So I am new to Vercel, and I was trying to deploy a template on the next.js website template section and this appeared: Deployment failed with an error. It failed to build. I do not know why.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/143552937/9d9fdff6-97a5-4f96-905a-0f4e98e403d7)

<html>
<body>

error - Failed to load next.config.mjs, see more info here https://nextjs.org/docs/messages/next-config-error
--
22:30:04.132 |  
22:30:04.132 | > Build error occurred
22:30:04.134 | Error: Invalid environment variables
22:30:04.134 | at f (file:///vercel/path0/node_modules/.pnpm/@t3-oss+env-nextjs@0.2.2_zod@3.21.4/node_modules/@t3-oss/env-nextjs/dist/index.mjs:1:505)
22:30:04.135 | at l (file:///vercel/path0/node_modules/.pnpm/@t3-oss+env-nextjs@0.2.2_zod@3.21.4/node_modules/@t3-oss/env-nextjs/dist/index.mjs:1:695)
22:30:04.135 | at C (file:///vercel/path0/node_modules/.pnpm/@t3-oss+env-nextjs@0.2.2_zod@3.21.4/node_modules/@t3-oss/env-nextjs/dist/index.mjs:1:868)
22:30:04.135 | at file:///vercel/path0/env.mjs:4:20
22:30:04.135 | at ModuleJob.run (node:internal/modules/esm/module_job:195:25)
22:30:04.135 | at async ModuleLoader.import (node:internal/modules/esm/loader:336:24)
22:30:04.135 | at async importModuleDynamicallyWrapper (node:internal/vm/module:429:15)
22:30:04.135 | at async loadConfig (/vercel/path0/node_modules/.pnpm/next@13.3.2-canary.13_@babel+core@7.21.4_@opentelemetry+api@1.1.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/server/config.js:528:36)
22:30:04.135 | at async Span.traceAsyncFn (/vercel/path0/node_modules/.pnpm/next@13.3.2-canary.13_@babel+core@7.21.4_@opentelemetry+api@1.1.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/trace/trace.js:103:20)
22:30:04.135 | at async /vercel/path0/node_modules/.pnpm/next@13.3.2-canary.13_@babel+core@7.21.4_@opentelemetry+api@1.1.0_react-dom@18.2.0_react@18.2.0/node_modules/next/dist/build/index.js:148:28
22:30:04.147 | ELIFECYCLE  Command failed with exit code 1.
22:30:04.163 | Error: Command "pnpm run build" exited with 1

</body>
</html>

This is the error

**Comment by swarnava** [comment]
Can you share the template link?

[a comment by an automated account is not shown]
